examples/usecases/incremental-training-with-layer-freezing.ipynb

```diff
@@ -0,0 +1,883 @@
+{
```

Comment:
<p>Can we add more descrption to</p><p> </p><p>"In this example, we fine-tune a model by setting different learning rates to different layers and freezing embedding tables."</p><p> </p><p>Why do we want to use different learning rates? Why do we freeze the embedding tables?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/809/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>